### ISSUE TITLE
Problems to build Caliburn.Micro

### ISSUE THREAD

**Issue by RudolfJan** [Caliburn-Micro/Caliburn.Micro#768]
I am trying to build the Caliburn.Micro project for the first time for Visual Studio 2019 Community Edition and this is not going very smoothly. It seems the application has not been completely rebuilt on a fresh environment. Probably it works all fine if you already built it in the past and then upgraded gradually. 

Some of the issues:

1. You need additional Visual Studio components, but it is a bit of guessing, because you need to build it for all platforms. I like to suggest to document the prerequisites in the documentation.
2. It may help to update the Nuget Packages references, not sure about that.
3. The Android an iOS versions seem a bit outdated. At least I tried installing newer versions. For Android I was required somehow to refer to Android 8.1 instead of 7.1 in the feature demo. Not sure if this is the correct step, because it is not documented what exactly the environment should be.

**Comment by vb2ae** [comment]
Here are the work flows I have installed.   You should not need the web workflows.  What errors are you getting?  

![image](https://user-images.githubusercontent.com/1483513/124681896-8c49b200-de97-11eb-82ab-a5a965fa5eab.png)

![image](https://user-images.githubusercontent.com/1483513/124681947-aaafad80-de97-11eb-9d2b-c8c47059151e.png)

![image](https://user-images.githubusercontent.com/1483513/124681988-c450f500-de97-11eb-860e-93ee5382e97b.png)

![image](https://user-images.githubusercontent.com/1483513/124682074-eea2b280-de97-11eb-8209-1e43cc424d28.png)

android
![image](https://user-images.githubusercontent.com/1483513/124682179-30335d80-de98-11eb-86f8-11ab7510e0c0.png)

![image](https://user-images.githubusercontent.com/1483513/124682244-53f6a380-de98-11eb-929a-ff6049f04ae4.png)

**Comment by RudolfJan** [comment]
Thanks, I managed to resolve all issues and got the feature demo running now. I also managed to compile the Caliburn.Micro project.

It took me about 8 hours, a bit trial and error to find out how to do it, but I did not document all steps, unfortunately. Also I am by no means sure my trial and error  creates a stable situation for the project.  I can do a pull request, but it would need very thorough review and maybe a significant amount of editing, so I only will do that if you think this is helpful. In short what I did:

 Updated all Nuget packages
Removed all references to Caliburn.Micro and got them again fro Nuget where needed
Removed and added a number of assembly references
Fixed one code issue in the Bootstrapper class 

```
protected override async void OnStartup(object sender, StartupEventArgs e)
{
    await DisplayRootViewForAsync(typeof(ShellViewModel));
}
```

Updated the minimum Android version to 10

and a lot I probably forget to mention. So most of it to get contextual conditions up to date again and one bug, which may not be a a bug,

At least I hope this helps and it now allows me to study the feature demo properly.
